src/Sylius/Bundle/ShopBundle/templates/account/address_book/create.html.twig

```diff
@@ -1,13 +1,13 @@
-{% extends '@SyliusShop/account/address_book/layout.html.twig' %}
+{% extends '@SyliusShop/shared/layout/base.html.twig' %}
 
-{% block title %}{{ 'sylius.ui.add_address'|trans }} | {{ parent() }}{% endblock %}
+{% set prefixes = ['sylius_shop.account.address_book.create', 'sylius_shop.shared.account'] %}
```

Comment:
I wonder if we shouldn't change `shared` keyword to `common` in entire shop to be more consistent with admin 🤔 
```suggestion
{% set prefixes = ['sylius_shop.account.address_book.create', 'sylius_shop.common.account'] %}
```